README.md

```diff
@@ -376,6 +378,28 @@ BigtableDataSettings.enableOpenCensusStats();
 BigtableDataSettings.enableGfeOpenCensusStats();
 ```
 
+You can view the metrics on the Google Cloud Platform Console
+[Metrics explorer](https://console.cloud.google.com/monitoring/metrics-explorer)
+page.
+
+You can configure how frequent metrics are pushed to StackDriver and the
```

Comment:
nit: may be nicer to inline these suggestions into the example below. People may read code more easily than documentation

esp. for monitored resource, we want to encourage users to choose their own instead of just going w/ global.